Dockerfile

```
@@ -0,0 +1,42 @@
+      -exec sh -c 'cp "$1" "${1%.example}"' _ {} \; \
+    && ./gradlew ":${APP_MODULE}:bootJar" --no-daemon \
+    && cp "${APP_MODULE}"/build/libs/*.jar /workspace/application.jar
+
+FROM eclipse-temurin:21-jre-jammy
+
+RUN useradd --system --uid 10001 --create-home app
+WORKDIR /app
+
+COPY --from=builder --chown=app:app /workspace/application.jar application.jar
+
+USER app
```

Hadolint (2.15.1)

[info] Non-numeric user-id may not be resolvable by host system

(DL3066)